packages/docusaurus-theme-classic/codeTranslations/he.json

```diff
@@ -0,0 +1,55 @@
+{
+  "theme.AnnouncementBar.closeButtonAriaLabel": "לסגור",
+  "theme.CodeBlock.copied": "הועתק",
+  "theme.CodeBlock.copy": "להעתיק",
+  "theme.CodeBlock.copyButtonAriaLabel": "להעתיק קוד",
+  "theme.NotFound.p1": "אנחנו לא מוצאים מה שאתה מנסה לחפש.",
+  "theme.NotFound.p2": "נא פנה למנהל האתר מאיפה קיבלת קישור הזה ותגיד לו שקישור לא תקין.",
+  "theme.NotFound.title": "דף לא נמצא",
+  "theme.Playground.liveEditor": "Live Editor",
+  "theme.Playground.result": "תוצאה",
+  "theme.PwaReloadPopup.closeButtonAriaLabel": "לסגור",
+  "theme.PwaReloadPopup.info": "קיימת גרסא חדשה",
+  "theme.PwaReloadPopup.refreshButtonText": "לרענן",
+  "theme.SearchBar.label": "חיפוש",
+  "theme.SearchPage.algoliaLabel": "חיפוש by Algolia",
+  "theme.SearchPage.documentsFound.plurals": "נמצא מסמך אחד|{count} מסמכים נמצאו",
+  "theme.SearchPage.emptyResultsTitle": "חפש בדוקומנטאציה",
+  "theme.SearchPage.existingResultsTitle": "תוצאות חיפוש עבור \"{query}\"",
+  "theme.SearchPage.fetchingNewResults": "מקבל תוצאות חדשות...",
+  "theme.SearchPage.inputLabel": "חיפוש",
+  "theme.SearchPage.inputPlaceholder": "הקלד כאן לחיפוש",
+  "theme.SearchPage.noResultsText": "לא נמצאו תוצאות",
+  "theme.blog.paginator.navAriaLabel": "רשימת דפים של הבלוג",
+  "theme.blog.paginator.newerEntries": "הכי חדש",
+  "theme.blog.paginator.olderEntries": "ישן יותר",
+  "theme.blog.post.paginator.navAriaLabel": "רשימת חדשות של הבלוג",
+  "theme.blog.post.paginator.newerPost": "הכי חדש",
+  "theme.blog.post.paginator.olderPost": "ישן יותר",
+  "theme.blog.post.plurals": "חדשה אחד|{count} חדשות",
+  "theme.blog.post.readMore": "עוד...",
+  "theme.blog.post.readingTime.plurals": "דקה לקריאה|{readingTime} דקות לקריאה",
```

Comment:
"he" has 4 plural forms, but this pluralized string only handle 2 plural forms

Reply:
One:
דקת לקריאה

Two:
שתי דקות קריאה
דקתיים means "2 minutes" but it's rarely used in this context

Many, other:
{readingTime} דקות קריאה

Notice - I changed the context a bit from [it takes] "4 minutes [in order] to read" to "4 minutes of reading" [noun]